utils_test.py

```diff
@@ -327,3 +327,13 @@ def test_get_supported_audio_language_description_with_invalid_code(self):
             Exception, 'Unsupported audio language code: invalid_code'):
             utils.get_supported_audio_language_description(
                 invalid_language_code)
+
+    def test_snake_case_to_camel_case(self):
+        snake_case_str1 = utils.snake_case_to_camel_case('user_id_number')
+        snake_case_str2 = utils.snake_case_to_camel_case('hello_world')
+        snake_case_str3 = utils.snake_case_to_camel_case('test1')
+        test_string1 = utils.snake_case_to_camel_case('user-id-number')
+        self.assertEqual(snake_case_str1, 'userIdNumber')
+        self.assertEqual(snake_case_str2, 'helloWorld')
+        self.assertEqual(snake_case_str3, 'test1')
+        self.assertNotEqual(test_string1, 'userIdNumber')
```

Comment:
Do a self.assertEqual with the actual value. Generally, favour the stricter test, so that we know that the string does not get mangled in some other way.

Reply:
Done.